nextflow/conf/04_executor_profiles.config

```
@@ -0,0 +1,64 @@
+// *****************
+// PROFILE SETTINGS
+// control how software is used (i.e. conda, docker, etc)
+// *****************
+
+// Nextflow Autometa Pipeline Configuration.
+/*
+ To choose one of the available profiles (standard, cluster, chtc) you must pass
+ in the the -profile argument. 
+ i.e. 
+ 
+ ```bash
+ nextflow run main.nf -profile slurm 
+ ```
+
+ You may also specify multiple profiles by separating their names with a comma.
+ i.e.
+ 
+ ```bash
+ nextflow run autometa.nf -profile standard,docker
+ ```
+
+ Note: Standard profile (running locally) is implictly used if no profile is specified by the user.
+*/
+
+
+profiles {
+  
+  standard {
+    // https://www.nextflow.io/docs/latest/executor.html#local
+    // simple profile that runs all processess locally
+    
+    process.executor = "local" 
+  }
+
+  basic_slurm {
```

Should this be changed to correspond to line 13?

```suggestion
  slurm {
```